### ISSUE TITLE
Is it maintained

### ISSUE THREAD

**Issue by rashthedude** [enki-com/react-search-input#142]
I was wondering if this package is still being maintained?

**Comment by mathieudutour** [comment]
not really. I'm not using it anymore so I'm not going to spend time on it. But I'd be happy to merge PRs!

**Comment by arojunior** [comment]
Hi. A possible solution here: #144
